pkg/rehearse/configmaps.go

```diff
@@ -135,8 +136,11 @@ func (c *ciOperatorConfigs) FixupJob(job *prowconfig.Presubmit, repo string) {
 // rehearsal jobs.
 func (c *ciOperatorConfigs) Create() error {
 	cm := &v1.ConfigMap{
-		ObjectMeta: metav1.ObjectMeta{Name: c.configMapName},
-		Data:       map[string]string{},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:   c.configMapName,
+			Labels: map[string]string{rehearseLabel: strconv.Itoa(c.prNumber)},
```

Comment:
Don't we want this to be local to the specific batch of jobs triggered by this run?

Reply:
The only purpose at the moment is to reduce watch traffic — jobs that match but were not created by this execution (a much smaller set) are ignored by `waitForJobs`.

The PR number was a quick way of achieving that. I could not think of a unique identifier that was (1) more specific and (2) this simple.

Reply:
Hash of the job spec would work, but sure